metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -94,12 +94,20 @@
     is_partitioning_column = 'YES'
     -- Filter out special partitions (https://cloud.google.com/bigquery/docs/partitioned-tables#date_timestamp_partitioned_tables)
     and p.partition_id not in ('__NULL__', '__UNPARTITIONED__')
+    and STORAGE_TIER='ACTIVE'
```

Comment:
[asking to learn] what does this do?

Reply:
It only list the active partitions and not those which are store for long term storage and even cost more to query.

Reply:
https://cloud.google.com/bigquery/docs/information-schema-tables#partitions_view